Cubical/Categories/Displayed/Constructions/Reindex.agda

```diff
@@ -0,0 +1,97 @@
+{-
+
+-}
+
+{-# OPTIONS --safe #-}
+--
+module Cubical.Categories.Displayed.Constructions.Reindex where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Data.Sigma
+import      Cubical.Data.Equality as Equality
+import      Cubical.Data.Equality.Conversion as Equality
+
+open import Cubical.Categories.Category.Base
+open import Cubical.Categories.Constructions.BinProduct
+  renaming (Fst to FstBP ; Snd to SndBP)
+open import Cubical.Categories.Functor
+
+open import Cubical.Categories.Displayed.Base
+open import Cubical.Categories.Displayed.Properties
+open import Cubical.Categories.Displayed.Functor
+open import Cubical.Categories.Displayed.Instances.Terminal hiding (intro)
+import      Cubical.Categories.Displayed.Reasoning as HomᴰReasoning
+open import Cubical.Categories.Displayed.Magmoid
+
+private
+  variable
+    ℓB ℓB' ℓC ℓC' ℓCᴰ ℓCᴰ' ℓD ℓD' ℓDᴰ ℓDᴰ' ℓE ℓE' ℓEᴰ ℓEᴰ' : Level
+
+module _
+  {C : Category ℓC ℓC'} (Cᴰ : Categoryᴰ C ℓCᴰ ℓCᴰ')
+  where
+
+  private
+    module C = Category C
+    module R = HomᴰReasoning Cᴰ
+
+    -- todo: generalize upstream somewhere to Data.Equality?
+    isPropEqHom : ∀ {a b : C.ob} {f g : C [ a , b ]}
+                → isProp (f Equality.≡ g)
+    isPropEqHom {f = f}{g} =
+      subst isProp (Equality.PathPathEq {x = f}{y = g}) (C.isSetHom f g)
+
+  open Categoryᴰ Cᴰ
+
+  reind' : {a b : C.ob} {f g : C [ a , b ]} (p : f Equality.≡ g)
+      {aᴰ : ob[ a ]} {bᴰ : ob[ b ]}
+    → Hom[ f ][ aᴰ , bᴰ ] → Hom[ g ][ aᴰ , bᴰ ]
+  reind' p = Equality.transport Hom[_][ _ , _ ] p
```

Comment:
I have some vague vibes to why using the identity type might be better than the path type, but what does this actually gain over the existing `reind`

Reply:
see Quotient.More. I was able to replace an ad hoc construction with this new `reind'` without transport hell